docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -1,10 +1,10 @@
 ---
 id: contributing_overview
-title:  Contributing to the Marketplace
-sidebar_label:  Contributing to the Marketplace
+title:  Marketplace Items Lifecycle
```

Comment:
There is also this part which is the marketplace lifecycle: https://docs.mia-platform.eu/docs/marketplace/overview_marketplace#marketplace-items-lifecycle
Maybe: `Create Marketplace Item`?